src/uucore/src/lib/features/fsext.rs

```diff
@@ -362,13 +362,16 @@ extern "C" {
     fn get_mount_info(mount_buffer_p: *mut *mut StatFs, flags: c_int) -> c_int;
 
     #[cfg(any(
-        target_os = "freebsd",
         target_os = "netbsd",
         target_os = "openbsd",
         all(target_vendor = "apple", target_arch = "aarch64")
     ))]
     #[link_name = "getmntinfo"] // spell-checker:disable-line
     fn get_mount_info(mount_buffer_p: *mut *mut StatFs, flags: c_int) -> c_int;
+
+    #[cfg(target_os = "freebsd")]
+    #[link_name = "getmntinfo@FBSD_1.0"] // spell-checker:disable-line
```

Comment:
could you please add a comment explaining the why?
thanks

Reply:
Done